eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -35,7 +35,6 @@ jobs:
       Linux_x64_gcc5_with_unit_test:
         Pool: azsdk-pool-mms-ubuntu-1804-general
         OSVmImage: MMSUbuntu18.04
-        VcpkgInstall: 'curl[ssl] libxml2 openssl'
```

Comment:
why you don't set cache mode here ? but you do earlier

Reply:
Because the mode is set in the initial ci configuration and it becomes an env var that is consumed by the vcpkg step. The rest of the ci configuration don't care about it.